lib/widget/input/dropdown.dart

```diff
@@ -99,6 +98,13 @@ abstract class SelectOne extends StatefulWidget
     return null;
   }
 
+  void _setValue(dynamic value) {
+    _controller.maybeValue = value;
```

Comment:
I know this is the existing code but it maybe better to check the validity here and either ignore to set the value (vs waiting for later). No more maybeValue.